android/src/main/kotlin/com/github/picture2pc/android/ui/main/MainScreen/elements/Clients.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.picture2pc.android.ui.main.MainScreen.elements
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.github.picture2pc.android.viewmodel.MainScreenViewModels.ClientsViewModel
+import org.koin.compose.rememberKoinInject
+
+@Composable
+fun Clients(modifier: Modifier){
```

Comment:
`ConnectedClientsList`. Also why does this even exist? The only thing we know about clients is their IP address. I would add a connection counter at most

Reply:
I thought the clients will have names too?